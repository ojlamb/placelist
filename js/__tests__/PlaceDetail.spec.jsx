import React from "react";
import Detail from "../components/places/PlaceDetail";

test("Detail renders correctly", () => {
  const component = shallow(<Detail />);
  expect(component).toMatchSnapshot();
});
