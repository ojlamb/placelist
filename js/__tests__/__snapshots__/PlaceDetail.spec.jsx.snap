// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail renders correctly 1`] = `
<Connect(PlaceDetail)
  classes={
    Object {
      "root": "Connect-PlaceDetail--root-1",
    }
  }
/>
`;
